File: exercises/beecrowd/1043/1043.go
// https://judge.beecrowd.com/pt/problems/view/1043
package main

import (
	"fmt"
)

func main() {
	var a, b, c float64

	fmt.Scanf("%f %f %f", &a, &b, &c)

	if a+b > c && a+c > b && b+c > a {
		fmt.Printf("Perimetro = %.1f\n", a+b+c)
	} else {
		fmt.Printf("Area = %.1f\n", ((a+b)*c)/2)
	}
}

File: exercises/beecrowd/1043/1043.java
// https://judge.beecrowd.com/pt/problems/view/1043
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        double a = sc.nextDouble();
        double b = sc.nextDouble();
        double c = sc.nextDouble();

        if (a < b + c && b < a + c && c < a + b) {
            double perimeter = a + b + c;
            System.out.printf("Perimetro = %.1f%n", perimeter);
        } else {
            double area = ((a + b) * c) / 2.0;
            System.out.printf("Area = %.1f%n", area);
        }

        sc.close();
    }
}
